### PR TITLE
Automated API of ticket assigned to me

### PR DESCRIPTION
**Pull request by ajit1865**
Automate the API's for feature tickets assigned to me.

### CODE REVIEW

**Review by deepansh257 — COMMENTED** [review]
Approved

**Comment by Sparsh79** [comment]
Approved.

File: go1percent/nightwatch.conf.js [hunk 1]
```diff
@@ -77,13 +78,10 @@ module.exports = {
...
-        browserName: 'chrome'
-        // 'goog:chromeOptions': {
-        //   ...
-        //   // specify the locateStrategy as xpath
-        //   locateStrategy: 'xpath',
-
-        // },
+        browserName: 'chrome',
+        chromeOptions: {
+           args: ['--headless', '--no-sandbox']
```
Comment (Ritika-gupta20) [inline go1percent/nightwatch.conf.js:83]: Why you have added headless mode in default config, we already have dedicated env settings for headless chrome. Let's keep the default env basic(without any custom args)

File: go1percent/nightwatch.conf.js [hunk 2]
```diff
@@ -102,7 +100,7 @@ module.exports = {
           acceptInsecureCerts: true,
           'moz:firefoxOptions': {
             args: [
-              // '-headless',
+              '-headless',
```
Comment (Ritika-gupta20) [inline go1percent/nightwatch.conf.js:103]: Why here have you set this to headless again? This is the ticket for API and headless mode is not required. There are no changes needed in this file it you have taken pull from the main branch

File: go1percent/package-lock.json
Comment (Ritika-gupta20) [inline go1percent/package-lock.json:1]: Have you installed any new dependency? If not then why have you committed this file here.

File: nightwatch.conf.js
Comment (Ritika-gupta20) [inline nightwatch.conf.js:1]: Why do we have two config file? Config should only exist inside go1percent directory

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]

**Review by Sparsh79 — APPROVED** [review]

File: go1percent/package-lock.json
Comment (ajit1865) [reply go1percent/package-lock.json:1]: i update this code from main so I commit it

File: nightwatch.conf.js
Comment (ajit1865) [reply nightwatch.conf.js:1]: Removed another config file

File: go1percent/nightwatch.conf.js (hunk 2 above)
Comment (ajit1865) [reply go1percent/nightwatch.conf.js:103]: my code throws some error when running via headless like chrome session so I made this changes

**Review by Ritika-gupta20 — APPROVED** [review]

**Review by deepansh257 — APPROVED** [review]

**Pull request merged**